sasdata/fair_database/data/models.py

```diff
@@ -166,26 +164,26 @@ class OperationTree(models.Model):
     )
 
 
-'''
 class Session(Data):
     """Database model for a project save state."""
 
+    # title
+    title = models.CharField(max_length=200)
+
     # dataset
-    # dataset = models.ManyToManyField(DataSet)
+    dataset = models.ManyToManyField(DataSet)
```

Comment:
Same question here, should we for simplicity say that a data set can only belong to one session? These are things we can discuss.